hoomd/mpcd/SphereGeometry.h

```diff
@@ -0,0 +1,195 @@
+
```

Comment:
I can't comment on it, but remove the extra file you added above ^^. It is `SphereGeometry.h` in the root directory, whereas this one is in the right place (`hoomd/mpcd`).